specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -2760,6 +2792,28 @@
       },
       "description": "The gallery artifact version source."
     },
+    "ExecutedValidation": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "This property specifies the type of image version validation."
+        },
+        "status": {
+          "type": "string",
+          "description": "This property specifies the status of the validationProfile of the image version."
+        },
+        "version": {
+          "type": "string",
+          "description": "This property specifies the valid version of the validation."
```

Comment:
How is this version used? Can only versions with this value be added to this profile?